src/components/Input/CheckBox/CheckBox.styles.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+import Checkmark from "../../Icons/Checkmark";
+import { constants, themes } from "../../../theme";
+
+export const CheckBoxWrapper = styled.label`
+  position: relative;
+  outline: none;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  text-align: left;
+  line-height: 1.5;
+  color: ${({ theme: { themeName } }) => themes[themeName].gray01};
+
+  &.checkbox--disabled {
+    cursor: default;
+    pointer-events: none;
+  }
+
+  &.checkbox--large {
+    font-size: 16px;
+  }
+
+  &.checkbox--small {
+    font-size: 14px;
```

Comment:
Can you add `font-size` constants here please?

Reply:
Sure. Thanks.